src/Microsoft.Health.Encryption/Customer/Health/CustomerKeyConstants.cs

```diff
@@ -0,0 +1,21 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using Azure;
+using Microsoft.Data.SqlClient;
+
+namespace Microsoft.Health.Encryption.Customer.Health;
+
+public static class CustomerKeyConstants
```

Comment:
Can't you define these as methods in their corresponding health check?

Reply:
DurableTaskHubHealthCheck will need the storage filter. and not sure if any checks for FHIR will need the SQL filter, but thought since storage is here, it made sense for both of them to be here

Reply:
Discussed offline, created new abstract classes to cover it